docs/cert-issuer/index.rst

```diff
@@ -0,0 +1,62 @@
+###########
+cert-issuer
+###########
+
+.. list-table::
+   :widths: 10,40
+
+   * - Edit on GitHub
+     - `/services/cert-issuer <https://github.com/lsst-sqre/lsp-deploy/tree/master/services/cert-issuer>`__
+   * - Type
+     - Helm_
+   * - Namespace
+     - ``cert-issuer``
+
+.. rubric:: Overview
+
+The ``cert-issuer`` application creates a cluster issuer for the use of the Rubin Science Platform.
+The issuer is named ``cert-issuer-letsencrypt-dns``.
+To configure an ingress to use certificates issued by it, add a ``tls`` configuration to the ingress and the annotation:
+
+.. code-block:: yaml
+
+   cert-manager.io/cluster-issuer: cert-issuer-letsencrypt-dns
+
+It depends on `cert-manager <https://cert-manager.io>`__.
+On most clusters where the Rubin Science Platform manages certificates, this is also handled by the Rubin Science Platform Argo CD, but on the base and summit clusters, cert-manager is maintained by IT and installed outside of Argo CD.
+
+.. rubric:: Bootstrapping the application
+
+The issuer defined in the ``cert-issuer`` application uses the DNS solver.
+The advantage of the DNS solver is that it works behind firewalls and can provision certificates for environments not exposed to the Internet, such as the Tucson teststand.
+
+The DNS solver uses an AWS service user with write access to Route 53 to answer Let's Encrypt challenges.
+To configure it, add the following to the ``values-*.yaml`` file for an environment:
+
+.. code-block:: yaml
+
+   solver:
+     route53:
+       aws-access-key-id: AKIAQSJOS2SFLUEVXZDB
+       hosted-zone: Z06873202D7WVTZUFOQ42
```

Comment:
This is maybe a little pedantic, but for some of these non-secret-secrets, what do you think about rolling them up into the vault secret?  I only ask because some of these things also have to have matching corresponding vault keys.  Not really an action for now, but maybe just a thought.

Reply:
Yeah, I wasn't entirely happy with that. Unfortunately, I'm not sure there's an option, since it doesn't look like a `ClusterIssuer` can get those values from secrets. They are supposed to be set directly in the resource. I did also put the access key into the Vault secret so that it's kept with the secret key.